testng-core/src/test/java/test/dependent/DependentTest.java

```diff
@@ -227,4 +206,84 @@ public void simpleCyclingDependencyShouldWorkWitParallelism(
                 + Arrays.deepToString(runMethods));
     assertThat(listener.getSucceedMethodNames()).containsExactlyInAnyOrder(runMethods[0]);
   }
+
+  @Test(description = "GITHUB-2658")
+  public void testMethodDependencyAmidstInheritance() {
+    TestNG testng = new TestNG();
+    testng.setTestClasses(new Class[] {PassingClassSample.class, FailingClassSample.class});
+    testng.setVerbose(2);
```

Comment:
Is verbose important for the test?

Reply:
Nah. Its not. It came in as part of a template `IssueTest` i use when I first build the unit tests :) Forgot to get rid of it, after I moved that test into this class. Basically a bone headed mistake 😁 
Fixed.